core/drivers/crypto/caam/hal/common/hal_ctrl.c

```diff
@@ -75,7 +75,7 @@ bool caam_hal_ctrl_splitkey_support(vaddr_t baseaddr)
 {
 	uint32_t val = io_caam_read32(baseaddr + CTPR_LS);
 
-	return !!GET_CTPR_LS_SPLIT_KEY(val);
+	return !GET_CTPR_LS_SPLIT_KEY(val);
```

Comment:
Oops! You left one `!`.

Reply:
Oops ... fixed

Reply:
note it's been fixed in the wrong commit.